dev/src/codewind/connection/UserProjectCreator.ts

```diff
@@ -273,6 +301,41 @@ namespace UserProjectCreator {
 
         // return bindRes;
     }
+
+    async function requestRemoteBindStart(connection: Connection, projectName: string,
```

Comment:
I would like to see a proper return type. It looks like the caller expects a project ID, so perhaps the return type should be `Promise<{ projectID: string }>`, plus any other keys that may exist. or create an interface if it's more complex